ruby/ql/test/query-tests/security/decompression-api/decompression_api.rb

```diff
@@ -0,0 +1,21 @@
+class TestController < ActionController::Base
+    # this should get picked up
+    def unsafe_zlib_unzip
+        Zlib::Inflate.inflate(params[:file])
+    end
```

Comment:
Let's write the test case like to to make sure we also find indirect uses of `params[..]`
```suggestion
    def unsafe_zlib_unzip
        path = params[:file]
        Zlib::Inflate.inflate(path)
    end
```